docs/index.asciidoc

```diff
@@ -23,14 +23,26 @@ include::{include_path}/plugin_header.asciidoc[]
 
 Read snmp trap messages as events
 
-Resulting `@message` looks like :
+Resulting `message` field looks like :
```

Comment:
```suggestion
Resulting `message` field resembles:
```

Reply:
I know that this text was already there, but we can use this opportunity to tweak it. I try to use "resembles" because it won't be exact.  Not that I think anybody would *really* get confused over this.  :-D  But more precise language is always better.